configs/swinir/swinir_x2_p48w8_d6e180_500k_div2k.py

```diff
@@ -0,0 +1,141 @@
+_base_ = [
```

Comment:
Rename config to the format of `{model}_[module setting]_{training schedule}_{dataset}` Like rename `swinir_x2_p48w8_d6e180_500k_div2k.py` to `swinir_x2p48w8d6e180_1xb8-500k_div2k.py`